kiwix-serve error: 'class kiwix::Server' has no member named 'setCatalogOnlyMode'

The current master branch is failing on kiwix-tools. Researching...

```cpp
2025-09-21T10:20:30 >>> kiwix-tools main Building
2025-09-21T10:20:30 GIT_DIR=. PATH="/opt/Lyra-SDK/buildroot/output/rockchip_rk3506_picocalc_luckfox/host/bin:/opt/Lyra-SDK/buildroot/output/rockchip_rk3506_picocalc_luckfox/host/sbin:/usr/lib/ccache:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin" FORCE_UNSAFE_CONFIGURE=1  PYTHONNOUSERSITE=y /opt/Lyra-SDK/buildroot/output/rockchip_rk3506_picocalc_luckfox/host/bin/ninja   -C /opt/Lyra-SDK/buildroot/output/rockchip_rk3506_picocalc_luckfox/build/kiwix-tools-main//build
2025-09-21T10:20:30 ninja: Entering directory `/opt/Lyra-SDK/buildroot/output/rockchip_rk3506_picocalc_luckfox/build/kiwix-tools-main//build'
[1/6] Compiling C++ object src/server/kiwix-serve.p/kiwix-serve.cpp.o
2025-09-21T10:20:31 FAILED: src/server/kiwix-serve.p/kiwix-serve.cpp.o
2025-09-21T10:20:31 /opt/Lyra-SDK/buildroot/output/rockchip_rk3506_picocalc_luckfox/host/bin/arm-buildroot-linux-gnueabihf-g++ -Isrc/server/kiwix-serve.p -Isrc/server -I../src/server -I/opt/Lyra-SDK/buildroot/output/rockchip_rk3506_picocalc_luckfox/host/arm-buildroot-linux-gnueabihf/sysroot/usr/include -I/opt/Lyra-SDK/buildroot/output/rockchip_rk3506_picocalc_luckfox/host/arm-buildroot-linux-gnueabihf/sysroot/usr/include/docopt -fdiagnostics-color=always -Wall -Winvalid-pch -Werror -std=c++17 -O3 '-DKIWIX_TOOLS_VERSION="3.8.0"' -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_TIME_BITS=64 -O2 -g0 -D_FORTIFY_SOURCE=1 -pthread -MD -MQ src/server/kiwix-serve.p/kiwix-serve.cpp.o -MF src/server/kiwix-serve.p/kiwix-serve.cpp.o.d -o src/server/kiwix-serve.p/kiwix-serve.cpp.o -c ../src/server/kiwix-serve.cpp
2025-09-21T10:20:31 ../src/server/kiwix-serve.cpp: In function 'int main(int, char**)':
2025-09-21T10:20:31 ../src/server/kiwix-serve.cpp:388:10: error: 'class kiwix::Server' has no member named 'setCatalogOnlyMode'
2025-09-21T10:20:31 388 |   server.setCatalogOnlyMode(catalogOnlyFlag);
2025-09-21T10:20:31 |          ^~~~~~~~~~~~~~~~~~
2025-09-21T10:20:31 ../src/server/kiwix-serve.cpp:391:10: error: 'class kiwix::Server' has no member named 'setContentServerUrl'
2025-09-21T10:20:31 391 |   server.setContentServerUrl(contentServerURL);
2025-09-21T10:20:31 |          ^~~~~~~~~~~~~~~~~~~
[3/6] Compiling C++ object src/manager/kiwix-manage.p/kiwix-manage.cpp.o
2025-09-21T10:20:31 ninja: build stopped: subcommand failed.
2025-09-21T10:20:31 make: *** [package/pkg-generic.mk:303: /opt/Lyra-SDK/buildroot/output/rockchip_rk3506_picocalc_luckfox/build/kiwix-tools-main/.stamp_built] Error 1
2025-09-21T10:20:31 make: Leaving directory '/opt/Lyra-SDK/buildroot'
Please check details in /opt/Lyra-SDK/output/sessions/2025-09-21_09-37-39/br-rockchip_rk3506_picocalc_luckfox_2025-09-21_09-39-31.log
ERROR: Running /opt/Lyra-SDK/device/rockchip/common/scripts/mk-rootfs.sh - build_buildroot failed!
ERROR: exit code 1 from line 16:
    "$RK_SCRIPTS_DIR/mk-buildroot.sh" $RK_BUILDROOT_CFG "$IMAGE_DIR"
ERROR: call stack:
    mk-rootfs.sh: build_buildroot(16)
    mk-rootfs.sh: build_hook(311)
    mk-rootfs.sh: main(340)
ERROR: Running /opt/Lyra-SDK/device/rockchip/common/build-hooks/99-all.sh - build_all failed!
ERROR: exit code 1 from line 21:
    "$RK_SCRIPTS_DIR/mk-rootfs.sh"
ERROR: call stack:
    99-all.sh: build_all(21)
    99-all.sh: build_hook(136)
    build-helper: try_func(63)
    build-helper: try_hook(96)
    build-helper: source(165)
    99-all.sh: main(147)
ERROR: Running /opt/Lyra-SDK/device/rockchip/common/build-hooks/99-all.sh - try_func build_hook all failed!
ERROR: exit code 1 from line 67:
    build_hook
ERROR: call stack:
    build-helper: try_func(67)
    build-helper: try_hook(96)
    build-helper: source(165)
    99-all.sh: main(147)
```

--- a/base/buildroot/package/kiwix-tools/kiwix-tools.mk
+++ b/base/buildroot/package/kiwix-tools/kiwix-tools.mk
@@ -4,7 +4,7 @@
 #
 ################################################################################
 
-KIWIX_TOOLS_VERSION = main
+KIWIX_TOOLS_VERSION = acad8a85ab4706ff527cabdaac4635a930f3bdd4
 KIWIX_TOOLS_SITE = $(call github,kiwix,kiwix-tools,$(KIWIX_TOOLS_VERSION))
 KIWIX_TOOLS_SOURCE = kiwix-tools-$(KIWIX_TOOLS_VERSION).tar.gz
 KIWIX_TOOLS_LICENSE = GPL-3.0+

